makePoplogCommander.sh

```diff
@@ -144,6 +232,13 @@ poplog env [OPTION]... [-] [NAME=VALUE]... [COMMAND [ARG]...]
 
         % poplog exec env [OPTION]... [-] [NAME=VALUE]... [COMMAND [ARG]...]
 
+poplog [NAME=VALUE]... [COMMAND [ARG]...]
+    Adds/modifies environment variables in the Poplog environment and
+    then processes the remainder of the arguments normally. This avoids
+    the necessity of invoking an 'env' process. Note: these bindings 
```

Comment:
~"this avoids the necessity of invoking an 'env' process."~
Don't think this adds much, the user doesn't care.

Reply:
I agree with the overall comment of removing the `env` action, which will make this section superfluous.

Reply:
Done